### PR TITLE
Move font loading to HTML.

### PR DESCRIPTION
**Pull request by XhmikosR**
Using `@import` is blocking.

Also remove the unneeded type `text/css` and use double quotes in meta charset tag.

BTW @parkr @benbalter @jasonlong  it seems there have been some changes in master that are not published/deployed. It would be nice if there was a new release.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Also, it seems there's a lot of duplicated CSS in print.css. Plus browsers download the print.css even when not printing. It would be better to use a media query and include the CSS in style.scss, but that's for another time.

File: _layouts/default.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <!DOCTYPE html>
 <html lang="{{ site.lang | default: "en-US" }}">
   <head>
-    <meta charset='utf-8'>
+    <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Chivo:900">
```
Comment (XhmikosR) [inline _layouts/default.html:7]: This could be tweaked later to include `display`.

https://addyosmani.com/blog/google-fonts-font-display/

**Comment by benbalter** [comment]
Thanks @XhmikosR! While this is a nice performance improvement, it would unfortunately be a breaking change for any user that has modified the default layout.

**Comment by XhmikosR** [comment]
@benbalter how are such cases handled though? Don't you ever needed to make breaking changes?

**Pull request closed without merging**